module_auto_update/models/module.py

```diff
@@ -17,49 +17,65 @@
 class Module(models.Model):
     _inherit = 'ir.module.module'
 
-    checksum_installed = fields.Char(
-        string="Installed Module Checksum",
-    )
     checksum_dir = fields.Char(
-        string="Current Module Checksum",
         compute='_compute_checksum_dir',
     )
-    upgrade_available = fields.Boolean(
-        default=False,
+    checksum_installed = fields.Char(
+        compute='_compute_checksum_installed',
```

Comment:
This field must not be computed. It only has to be updated when addon gets updated too, but it is a normal char field.